src/test/java/de/szut/lf8_project/lf8/project/DeleteIT.java

```diff
@@ -15,4 +26,16 @@ public void DeleteProject() throws Exception {
                  delete(String.format("/lf8/project/delete/%d", entity.getId())))
                 .andExpect(status().is2xxSuccessful());
     }
+
+    @Test
+    @WithMockUser(roles = "user")
+    public void DeleteEmployeeFromProject() throws Exception {
+        Set<Long> employees =  new HashSet<Long>();
+        employees.add(Integer.toUnsignedLong(1));
+        ProjectEntity ent1 = projectRepository.save(new ProjectEntity(2L, 3L, 4L, 5L, employees, new HashSet<>(), "Hallo Arad", LocalDate.now(), LocalDate.now().plusDays(2)));
+
+        this.mockMvc
+                .perform(delete(String.format("/lf8/project/%d/delete/employee/1", ent1.getId())))
+                .andExpect(status().is2xxSuccessful());
+    }
```

Comment:
Du solltest in dem Test auch nach schauen ob der Employee tatsächlich entfernt wurde und nicht einfach darauf vertrauen, dass es funktioniert hat, weil du den HTTP Code 200 bekommen hast.
Zudem solltest du auch noch Teste einbauen, welche überprüfen ob der Endpunkt richtig reagiert, sollte das Projekt oder der Mitarbeiter nicht gefunden werden